### PR TITLE
Implement Gulp plugin for TestCafe

### PR DESCRIPTION
**Pull request by inikulin**
\cc @VasilyStrelyaev @AndreyBelym @AlexanderMoskovkin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+
+Specifies the reporter.
+
+#### filter
+
+*Type*: `function(testName, fixtureName, fixturePath)`
+
+*Default*: `null`
+
+Allows you to manually select which tests should be run.
```
Comment (VasilyStrelyaev) [inline README.md:60]: Let's add more references to the docs from here.

Probably, add a section after the descriptions like

```
*Details*: (link)
```

Comment (VasilyStrelyaev) [reply README.md:60]: In this case,

```
*Details*: [runner.filter](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#filter)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+
+Defines whether to enable the [quarantine mode](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#quarantine-mode).
+
+#### selectorTimeout
+
+*Type*: `Number`
+
+*Default*: `10000`
+
+Specifies the amount of time, in milliseconds, within which selectors make attempts to obtain a node to be returned.
```
Comment (VasilyStrelyaev) [inline README.md]: Maybe add a link to selectors?

```
Specifies the amount of time, in milliseconds, within which [selectors](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html) make attempts to obtain a node to be returned.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,101 @@
...
+
+The path to which the screenshots will be saved. Enables the test runner to take screenshots of the tested webpages.
+
+#### takeScreenshotsOnFail
+
+*Type*: `Boolean`
+
+*Default*: `false`
+
+Specifies if screenshots should be taken automatically whenever a test fails.
```
Comment (VasilyStrelyaev) [inline README.md]: Let's give a link to docs... or just add that you should specify the path.

```
Specifies if screenshots should be taken automatically whenever a test fails. Requires that the [screenshotsPath](#screenshotsPath) is set.
```

**Review by VasilyStrelyaev — CHANGES_REQUESTED** [review]
\r-

File: test/manual/mocha-test.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+it('Should pass', function () {
```
Comment (AndreyBelym) [inline test/manual/mocha-test.js:1]: Everything is fine, except that I didn't get the purpose of this file (and `unit` task in `Gulpfile`), sorry.

Comment (inikulin) [reply test/manual/mocha-test.js:1]: It's for the manual testing as directory title suggests

**Review by AlexanderMoskovkin — APPROVED** [review]
lgtm

**Review by AndreyBelym — APPROVED** [review]
lgtm

**Comment by inikulin** [comment]
FPR

**Review by VasilyStrelyaev — APPROVED** [review]
lgtm

**Pull request merged**